opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -3066,6 +3066,7 @@ enum OpinionsOrdering {
   createdBy
   objectMarking
   x_opencti_workflow_id
+  confidence
```

Comment:
yes, we added this to fix a current bug on production. If you try to order opinions on confidence, it will fail with an error :  
https://demo.opencti.io/dashboard/analysis/opinions?sortBy=created&orderAsc=false&searchTerm=&filters=%7B%22confidence_lte%22%3A%5B%7B%22id%22%3A%2250%22%2C%22value%22%3A%2250%22%7D%5D%7D&deSelectedElements=null

Reply:
Yes I understand for the filter, but where it's possible to order ?

Reply:
You can order a confidence in Sightings for exemple :)

Reply:
Oh, nice catch ! I missed this one.
So, if you add it here we need to add it in the UI.
If not, it's not necessary to add it on graphQL